NSS-Day-Cohort-55/tabloidfullstack-adequately-powerful-ibexes
# Fixing register path bug

# Description
Bug in path to register page on login page is fixed.


# Testing Instructions

1. Be logged out
2. Go to login page
3. Click "Register"
4. Verify that the registration form now appears
